### DISCUSSION TITLE
Images

### DISCUSSION THREAD

**Discussion by Kovalskiii** [rnmapbox/maps#2914] [General]
Hello, can somebody explain how exactly should it work, where do i need to indicate image source?
I want to add image for SymbolLayer.
```
<Mapbox.Images>
  <Mapbox.Image
     name="image-for-a-symbo-layer"
  >
<View style={{width:20, height:20 ,backgroundColor:'green', borderRadius: 10}}
/>
  </Mapbox.Image>
</Mapbox.Images>
<Mapbox.SymbolLayer 
  id='a-symbo-layer'
  style={{iconImage: 'image-for-a-symbo-layer'}}
/>
```

**Comment by mfazekas** [comment]
@Kovalskiii the code should work as you've shown. What's the issue?

**Reply by Kovalskiii** [reply]
I do  not understood how to indicate image. For example if i have (import eventImg from './src/assets/images/event.png')

**Reply by mfazekas** [reply]
So you use `Image` embedded in `Images` to define image with react-native drawing.  And use the `images` property to define images with url-s
```tsx
<Images 
   images={{'image-for-a-symbo-layer' : require('./src/assets/images/event.png')}}
/>
```
